README.md

```diff
@@ -1,74 +1,82 @@
 # Hello, world! 👋
-I am Fabricio Pashaj. I'm 16 year old and I'm a coding addict. I started to learn HTML by the middle of February, 2020.My favourite editor is Vim (just Vim, without the `N` at the begining).
+
+I am Fabricio Pashaj, a 16 year old coding addict. I started to learn HTML by the middle of February, 2020. My favourite editor is Vim (just Vim, without the `N` at the begining).
+
+## Stuff about me
+
+- I code on an android phone 📱.
+- I don't like GUIs.
+- I don't write cross-platform C code unless I have or someone asks me to.
+- I use gruvbox theme.
+- I like math, physics, biology and chemistry.
+- I like reading books but I hate literature.
```

Comment:
What does this mean?

Reply:
I don't write cross-platform C code unless [I have/someone tells me] to

Reply:
Sorry, I meant the 
> I like reading books but I hate literature.

part.

Reply:
literature == ELA for you && people may suppose that I like literature if I like reading books (imho)

Reply:
I see, you may want to put `I like reading books but I hate literature class.`